### ISSUE TITLE
[NODE] Bad Handshake with HTTP 200 using Geth node

### ISSUE THREAD

**Issue by DeflateAwning** [smartcontractkit/chainlink#4023]
**Description**
I run: `cd ~/.chainlink && docker run -p 6688:6688  -v ~/.chainlink:/chainlink -it --env-file=.env smartcontract/chainlink:0.10.1 local n`

I get the following result:
```
2021-03-03T06:45:52Z [DEBUG] Unlocked OCR key                                   offchainreporting/keystore.go:72
2021-03-03T06:45:52Z [INFO]  API exposed for user xxx@xxxx.com            cmd/local_client.go:103
2021-03-03T06:45:52Z [DEBUG] eth.Client#Dial(...)                               eth/client.go:128
error starting app: websocket: bad handshake (HTTP status 200 OK)
```

Ideally, a more descriptive error message would be very helpful.

**Basic Information**
- Network: Ethereum Mainnet
- Blockchain Client: Geth 1.9.25-stable
- Operating System: macOS 10.15.7
- Hosting Provider: self-hosted
- Startup Command: See above

**Comment by PatrickAlphaC** [comment]
Thanks for this issue. Adding.

**Comment by ThePiGuy** [comment]
Any idea what was causing this, or updates on this issue? Thanks!

**Comment by PatrickAlphaC** [comment]
Often times it's from a bad ETH 1 connection, or just anytime a handshake has a hard time going through.

**Comment by ThePiGuy** [comment]
That error happened repeatedly. Can someone please try to reproduce?

**Comment by PatrickAlphaC** [comment]
Ah ok, finally got it. Could you try on 10.5? I was able to get rid of the bug on 10.5

**Comment by PatrickAlphaC** [comment]
Closing for now... This I think has been fixed in 1.0.0
